### ISSUE TITLE
Build a feed-forward neural network

### ISSUE THREAD

**Issue by wetherc** [riffatahmad/Data-Science-Project#5]
**Estimated time:** 4 hours (might vary depending on the framework that you use)

Build a simple feed-forward neural network to predict the same as #4. To keep things simple, let's use the following architecture:

- 2 hidden layers of [512, 256] neurons
- sigmoid activation functions
- binary crossentropy loss
- adam optimizer
- dropout in both hidden layers with a 0.5 probability
- train for 30 epochs

**Comment by wetherc** [comment]
Note that you will need to one-hot encode any categorical columns in your dataset
